tests/Doctrine/ODM/MongoDB/Tests/Aggregation/Stage/SetTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ODM\MongoDB\Tests\Aggregation\Stage;
+
+use Doctrine\ODM\MongoDB\Aggregation\Stage\Set;
+use Doctrine\ODM\MongoDB\Tests\Aggregation\AggregationTestTrait;
+use Doctrine\ODM\MongoDB\Tests\BaseTest;
+
+class SetTest extends BaseTest
+{
+    use AggregationTestTrait;
+
+    public function testSetStage(): void
+    {
+        $setStage = new Set($this->getTestAggregationBuilder());
+        $setStage
+            ->field('product')
+            ->multiply('$field', 5);
+
+        self::assertSame(['$set' => ['product' => ['$multiply' => ['$field', 5]]]], $setStage->getExpression());
+    }
+
+    public function testProjectFromBuilder(): void
```

Comment:
```suggestion
    public function testSetFromBuilder(): void
```

On that note, I think there's a similar typo in AddFieldsTest, from which this was copied.